src/components/DataSetSelect/DataSetSelect.test.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { Provider } from 'react-redux';
+import { RootState } from 'reducers';
+import { MemoryRouter } from 'react-router-dom';
+import * as PermissionsUtils from 'utils/PermissionsUtils';
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { Select, Popover } from 'antd';
+import DataSetSelect from './DataSetSelect';
+
+jest.mock('utils/PermissionsUtils');
+
+configure({ adapter: new Adapter() });
```

Comment:
NIT: I put this in the jest setup file in datastudio, I think this will be needed for all enzyme react tests?

Reply:
good point lets do that!